packages/shared/lib/services/sync/config/config.service.ts

```diff
@@ -39,6 +41,15 @@ const convertSyncConfigToStandardConfig = (syncConfigs: extendedSyncConfig[]): S
 
         const syncName = syncConfig.sync_name;
 
+        const endpoint =
+            !syncConfig.endpoints_object || syncConfig?.endpoints_object?.length === 0
+                ? null
+                : syncConfig.endpoints_object.map((endpoint) => `${endpoint.method} ${endpoint.path}`);
+
+        if (!endpoint || endpoint.length === 0) {
+            isV1 = true;
```

Comment:
no endpoint necessarily means v1? I had the impression it was available on v1 too

Reply:
Good question. I just checked and if it is found to be v1 the endpoint isn't available.